joomla/joomla-cms
# [4.0] version number

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/26373.

### Summary of Changes
~~I changed the position of the module that displayed the version number to the bottom.~~ 
I changed the position of the module that displayed the version number. See https://github.com/joomla/joomla-cms/pull/26447#issuecomment-538655489

### Testing Instructions
1. Apply this patch.
2. Make a new installation (Have to update SQL in order for the patch to work).
3. See, that the version number is now ~~on the right side of the bottom~~  at the top right corner.


4. See, that no version number is displayed in the top menu.


### Documentation Changes Required
No
